react-homework/src/components/ItemCard/ItemCard.jsx

```diff
@@ -1,24 +1,49 @@
-import React from 'react';
-import Button from '../Button/Button';
+import React, { useRef, useContext } from 'react';
 import './ItemCard.css';
+import Button from '../Button/Button';
+import { useCart } from '../CartContext/CartContext';
 
 const ItemCard = ({ item }) => {
-    return (
-        <div className="item-card">
-            <img src={item.imageUrl} alt={item.name} className="item-image" />
-            <div className="item-info">
-                <div className="item-top">
-                    <h3>{item.name}</h3>
-                    <span className="item-price">$ {item.price} USD</span>
-                </div>
-                <p>{item.description}</p>
-                <div className="item-bottom">
-                    <input type="number" defaultValue="1" className="item-quantity" />
-                    <Button label='Add to card' isActive={true}></Button>
-                </div>
-            </div>
+  const { addToCart } = useCart();
+  const quantityInput = useRef(null);
+
+  const handleAddToCart = () => {
+    const quantity = parseInt(quantityInput.current.value || 0);
+    addToCart(quantity);
+  };
+
+  const handleChange = (event) => {
```

Comment:
It would be better to use more obvious name, because we can handle change many things on a project. E.g. handleChangeItemQty